### DISCUSSION TITLE
ANOVA and MANOVA support

### DISCUSSION THREAD

**Discussion by cdrnet** [mathnet/mathnet-numerics#843] [Ideas]
From: https://mathnetnumerics.codeplex.com/workitem/5704

> I am looking to implement a multivariant ANOVA (MANOVA) with some help (I am not a Statistician, just a Biologist who plays one ;D ). Can we add this to Math.NET easily? And if so what are the rules for code creation, et al. that I need to follow? Also I will probably implement One and 2 Way ANOVAS also (they should be easier).

**Comment by serialkiller** [comment]
Any update on this feature ? It would be really helpful to get it.

**Comment by ghost** [comment]
Any update? It would be very useful

**Comment by cdrnet** [comment]
Unfortunately I'm not aware of anyone actively working on this at the moment.

**Comment by DanielsCode** [comment]
any updates?

**Comment by millikens** [comment]
I would very much appreciate this as well.  I would be willing to work with someone on it!
